draft-dcook-ppm-dap-interop-test-design.md

```diff
@@ -92,7 +92,8 @@ Each of these test APIs should return a status code of 200 OK if the command was
 In multiple APIs defined below, the test runner will send the name of a VDAF, along with the parameters necessary to fully specify the VDAF. These will be stored in a nested object, with the following attributes (new `type` values and new keys will be added as new VDAFs are defined).
 
 |Key|Value|
-|`type`|One of `"Prio3Aes128Count"`, `"Prio3Aes128Sum"`, or `"Prio3Aes128Histogram"`|
+|`type`|One of `"Prio3Aes128Count"`, `"Prio3Aes128CountVec"`, `"Prio3Aes128Sum"`, or `"Prio3Aes128Histogram"`|
+|`length` (only present if `type` is `"Prio3Aes128CountVec`")|The length of the vector, used to parameterize the Prio3Aes128CountVec VDAF.|
```

Comment:
Nit: we should explicitly specify that the length is a JSON number. A plain number should be sufficient, since the length is a cardinal number. (same as "bits" for Prio3Aes128Sum)

Reply:
On the other hand, if we wanted to be more uniform, we could stringify all VDAF parameters.

Reply:
I think both these choices are good and I'm willing to do either. As the author of this document, do you have a preference?

Reply:
Let's make all VDAF parameters strings. That will keep things the simplest at the specification level, and we won't have to reason about expected ranges as new VDAFs are introduced.